ceph: Update pg_num related documents (#5247)
rook/rook: Documentation/ceph-common-issues.md

* Ceph: A fix that uses the above-mentioned kernel's feature. The Ceph community will probably discuss this fix after releasing Linux v5.6.

You can bypass this problem by using ext4 or any other filesystems rather than XFS. Filesystem type can be specified with `csi.storage.k8s.io/fstype` in StorageClass resource.

> ## Too few PGs per OSD warning is shown

Reviewer: Please add this to the table of contents links at the top of this topic so it is more discoverable

Author: Added, thanks.

### Symptoms

> - `ceph status` shows "too few PGs per OSD" warning.

Reviewer: How about showing a snippet of the output from `ceph status` in a multi-line comment so it's clear that this is a ceph health warning?

Author: Added the result of `ceph status` as you said.

### Solution

The meaning of this warning is written in [the document](https://docs.ceph.com/docs/master/rados/operations/health-checks#too-few-pgs).

> However, in many case, it's venign. For more information, please see [the blog entry](http://ceph.com/community/new-luminous-pg-overdose-protection/).

Reviewer: ```suggestion
However, in many case, it's benign. For more information, please see [the blog entry](http://ceph.com/community/new-luminous-pg-overdose-protection/).
```

Author: Ah, thank for finding my typo. Fixed.

> Please refer to [Configuring Pools](ceph-advanced-configuration.md#configuring-pools) if you want to know the proper pg_num of pools and change these values. 

Reviewer 2: ```suggestion
Please refer to [Configuring Pools](ceph-advanced-configuration.md#configuring-pools) if you want to know the proper `pg_num` of pools and change these values. 
```

Author: Fixed, thanks.

* Ceph: A fix that uses the above-mentioned kernel's feature. The Ceph community will probably discuss this fix after releasing Linux v5.6.

You can bypass this problem by using ext4 or any other filesystems rather than XFS. Filesystem type can be specified with `csi.storage.k8s.io/fstype` in StorageClass resource.

## Too few PGs per OSD warning is shown

### Symptoms

- `ceph status` shows "too few PGs per OSD" warning as follows.

```console
# ceph status
  cluster:
    id:     fd06d7c3-5c5c-45ca-bdea-1cf26b783065
    health: HEALTH_WARN
            too few PGs per OSD (16 < min 30)
```

### Solution

The meaning of this warning is written in [the document](https://docs.ceph.com/docs/master/rados/operations/health-checks#too-few-pgs).

> However, in many case, it's benign. For more information, please see [the blog entry](http://ceph.com/community/new-luminous-pg-overdose-protection/).

Reviewer: ```suggestion
However, in many cases it is benign. For more information, please see [the blog entry](http://ceph.com/community/new-luminous-pg-overdose-protection/).
```

Author: Fixed, thanks.
